frontend/src/scenes/experiments/charts/funnel/StepBars.tsx

```diff
@@ -20,8 +20,8 @@ export function StepBars({ step, stepIndex }: StepBarsProps): JSX.Element {
                     />
                 ))}
             </div>
-            {step.nested_breakdown?.map((series) => (
-                <StepBar key={`bar-${stepIndex}-${series.order}`} step={series} stepIndex={stepIndex} />
+            {step.nested_breakdown?.map((series, i) => (
```

Comment:
Although valid, using indexes as keys can cause reconciliation issues when re-rendering components. It's a very deep cut React thing, and most of the time it gets overlooked. There may be other better values, maybe the `breakdown_value`.
But we use indexes all the time, so this is fine. Keep calm and carry on.